### ISSUE TITLE
Is this tool still being mantained? 

### ISSUE THREAD

**Issue by ghevge** [swarmpit/swarmpit#663]
Is this tool still being mantained? 
I can see the last releas was back in 2020. 
Any plans for releasing the 2.0 version ?  I can see a feature in there that sounds apealing to me: service autoscaling

Thanks

**Comment by mrspartak** [comment]
Autoscaling requires some logic, and most probably CPU autoscaling will be not enough. You can write your own autoscaler that will either use docker socket or swarmpit API to scale pods depending on your own logic.

**Comment by suculent** [comment]
Indeed, we're happily using Swarmpit 1.9 at THiNX.cloud (well, like once a year the DB did get broken and Registry connections had to be reconfigured manually from scratch).

We're autoscaling the service from within our application using Docker Swarm. It's a bit of coding, but at least things are under control.

**Comment by mpcref** [comment]
Can someone give us a status update about the Swarmpit project? I see commits on master as recent as April of this year but the latest release stems from 2020. It's a very useful tool if everything works as intended but due to issues #605 and #676 it's unusable. Will there ever be a release where these issues are fixed or should we move on from Swarmpit?

**Comment by nohaapav** [comment]
@mpcref We spoke with @mrq1911 2 weeks ago and we'll invest some time to fix critical issues and release new version of swarmpit hopefully till the end of the year.

**Comment by mrspartak** [comment]
@nohaapav that's really great news!

**Comment by lepiroupo** [comment]
@nohaapav  Any updates about this new version? Tks!! :)

**Comment by nohaapav** [comment]
> @nohaapav Any updates about this new version? Tks!! :)

Didn't get to it yet. Still on the TO-DO list though ..
